.github/workflows/build-deploy-frontend.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       - name: Upload to Azure blob storage
         shell: bash
         run: |
-          az storage blob upload-batch --account-name reportvisionfrontenddev -d '$web' -s client-build/
+          az storage blob upload-batch --account-name reportvisionfrontenddev -d '$web' -s client-build/ --overwrite
```

Comment:
bit of scope creep here but missed this little detail on my last pr. this allows us to overwrite all the blob files every time we deploy, making sure deployments change everything from scratch. currently, if theres now id change it will on individual files it will skip it.